.github/workflows/ci.yml

```
@@ -0,0 +1,186 @@
+name: CI
+
+on:
```

```suggestion
on: [push, pull_request]
```

Probably it's a good idea not to limit running ci workflows only on a master branch since we have maint-0.* branches as well

---

Done 👍